### ISSUE TITLE
Support rollup 3

### ISSUE THREAD

**Issue by bug-tape** [asyncLiz/rollup-plugin-minify-html-literals#25]
Currently dependency doesn't allow working with rollup 3. I don't know if code must be changed too.

**Comment by Frank3K** [comment]
Work-around:

```js
import minifyHTML from 'rollup-plugin-minify-html-literals';
...
plugins: [
  ...,
  minifyHTML.default(), // instead of minifyHTML(),
  ...,
]
```

**Comment by kobleistvan** [comment]
Bump

**Comment by qeleb** [comment]
please?

**Comment by asyncliz** [comment]
Sorry for the delay all, I merged #26 to add support. I'll try to work on a release this week. Would love maintainer volunteers!

**Comment by bencbradshaw** [comment]
Hi @asyncLiz , looking forward to this release! Any way I could help out with it?
